android/src/ti/identity/FingerPrintHelper.java

```diff
@@ -272,4 +290,22 @@ public KrollDict deviceCanAuthenticate(int policy)
 		}
 		return response;
 	}
+
+	private void startDeviceCredentials()
+	{
+		KrollDict response = new KrollDict();
+		response = deviceCanAuthenticate(TitaniumIdentityModule.AUTHENTICATION_POLICY_BIOMETRICS);
```

Comment:
You need to pass in the policy setting returned by the `TitaniumIdentityModule.getAuthenticationPolicy()` method. Passing the module instance to `FingerPrintHelper` constructor will give you access to it.

The idea is we do NOT want to prompt the end-user for a passcode if the policy is set to `AUTHENTICATION_POLICY_BIOMETRICS`. That is, the app developer needs to opt-in to passcode support by setting the policy to `AUTHENTICATION_POLICY_PASSCODE` by setting it via the module.

Reply:
Done as suggested